components/list/list-item-mixin.js

```diff
@@ -38,9 +38,12 @@ function addTabListener() {
 	});
 }
 
+const listItemInteractiveFlag = window.D2L?.LP?.Web?.UI?.Flags.Flag('GAUD-7495-list-interactive-content', true) ?? true;
+
 let hasDisplayedKeyboardTooltip = false;
 
 export function isInteractiveInListItemComposedPath(e, isPrimaryAction) {
+	if (!listItemInteractiveFlag) return false;
```

Comment:
Just doing an early exit from this check to always do the default primary click action of the list items when the flag is off.